src/main/rules/GCI96/python/GCI96.asciidoc

```diff
@@ -1,3 +1,24 @@
+<<<<<<< HEAD
```

Comment:
revert your modifications on all GCI96 file ... because the current PR is GCI107 and not GCI96